### DISCUSSION TITLE
Build BeautifulNepal Map for Visitors With Accurate Nepali Geo Datas 🗺

### DISCUSSION THREAD

**Discussion by jsdhami** [TourismKit/beautifulnepal.org#5] [Ideas]
# BeautifulNepal Map

## Introduction

The BeautifulNepal website aims to also provide a comprehensive and interactive map for visitors to explore the scenic beauty of Nepal. In this document, we'll discuss the process of building the map and incorporating road data to enhance the user experience.

## Goals

1. **Interactive Map**: Create a user-friendly map that allows visitors to explore different regions of Nepal.
2. **Road Data Integration**: Incorporate road data to help users plan their journeys and navigate easily.

## Tools and Technologies

To achieve our goals, we'll leverage the following tools and technologies:

- **Leaflet.js**: A lightweight JavaScript library for interactive maps.
- **OpenStreetMap**: A collaborative mapping project that provides free, editable maps.
- **GeoJSON**: A format for encoding a variety of geographic data structures.

## Steps
- Will added..
-  Map Setup..

**Comment by devsdenepal** [comment]
I am also thinking the same thing, and we need to work on collecting data.
I am inspired from this repo: <a href="https://github.com/mesaugat/geoJSON-Nepal/"> geoJSON-Nepali</a>
